tests/virt_autotest/host_upgrade_step2_run.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+use base "host_upgrade_base";
+#use virt_utils qw(set_serialdev);
+use testapi;
+use strict;
+
+sub get_script_run() {
+    my $self = shift;
+
+    my $pre_test_cmd = $self->get_test_name_prefix;
+    $pre_test_cmd .= "-run 02";
+
+    return "$pre_test_cmd";
+}
+
+sub run() {
+    my $self = shift;
+
+    # Set the correct serial dev for ipmi xen and non-xen host according to the installed product release
+    # &virt_utils::set_serialdev();
+
+    # Got script run according to different kind of system
+    my $pre_test_cmd = $self->get_script_run();
+
+    # Execute script run
+    my $ret = $self->execute_script_run($pre_test_cmd, 36000);
```

Comment:
10 hours? Every test job is canceled after a "MAX_JOB_TIME" of 2h so this won't work